packages/front-end/styles/global.scss

```diff
@@ -556,6 +556,12 @@ pre {
 
 .gb-tooltip {
   border-radius: 5px;
+  white-space: normal;
+  color: var(--text-color-muted);
+  font-size: 13px;
+  text-transform: initial;
+  font-weight: normal;
+  font-style: italic;
```

Comment:
We use tooltips for a lot of different things.  I like the idea of resetting the font-weight, font-size, text-transform, and text-weight.  But, not sure about setting color to muted and font-style to italic.  That seems like a specific style choice that we would only want to make some of the time.

Tooltip bodies can contain React components, so it might make sense to default to a normal font-color and style and then a specific tooltip could wrap the body in `<em className="text-muted">` if desired.

Reply:
sure, that makes sense. I removed those features. Decided against adding italics and muted color using <em> as other tooltips don't seem to follow this. Might be good to take inventory of these and other reusable components at some point.